src/app/components/top-nav.tsx

```diff
@@ -0,0 +1,72 @@
+"use client";
+import { Button } from "@/components/ui/button";
+import { Separator } from "@/components/ui/separator";
+import {
+  Sheet,
+  SheetContent,
+  SheetHeader,
+  SheetTrigger,
+} from "@/components/ui/sheet";
+import { cn } from "@/lib/utils";
+import { Bars3Icon, HomeIcon } from "@heroicons/react/20/solid";
+import Link, { LinkProps } from "next/link";
+import { usePathname } from "next/navigation";
+import { useState } from "react";
+
+type MobileLinkProps = LinkProps & {
+  children: React.ReactNode;
+  className?: string;
+  onOpenChange: (open: boolean) => void;
+};
+function MobileLink({ onOpenChange, className, ...props }: MobileLinkProps) {
+  const pathname = usePathname();
+  const isActive = pathname === props.href;
+  return (
+    <Link
+      aria-current={isActive ? "page" : undefined}
+      className={cn(
+        "flex flex-row items-center w-full rounded px-4 py-2 aria-[current='page']:bg-gray-100",
+        className
+      )}
+      onClick={() => onOpenChange(false)}
+      {...props}
+    />
+  );
+}
+
+function MobileNav() {
+  const [open, setOpen] = useState(false);
+  return (
+    <Sheet open={open} onOpenChange={setOpen}>
+      <SheetTrigger asChild>
+        <Button size="icon" variant="ghost">
+          <Bars3Icon className="size-5" />
+          <span className="sr-only">menu</span>
+        </Button>
+      </SheetTrigger>
+      <SheetContent side="left">
+        <h2 className="text-xl font-bold">Birds</h2>
+        <nav className="mt-3">
+          <ul>
+            <li>
+              <MobileLink onOpenChange={setOpen} href="/">
+                <HomeIcon className="size-4 mr-2" /> Home
+              </MobileLink>
+            </li>
+          </ul>
+        </nav>
+      </SheetContent>
+    </Sheet>
+  );
+}
+
+export function TopNav() {
+  return (
+    <div className="sticky top-0 bg-white z-10">
```

Comment:
`z-10` is important here 'cause of how we load images. If we don't set a `z-index` then the images appear on top of the navigation and it looks _weird_:

<table>
  <tr>
    <td>without z-index
    <td>with z-index
  <tr>
    <td><img width="418" alt="image" src="https://github.com/user-attachments/assets/e3303424-5589-4cb6-96eb-37307337d6a8">
    <td><img width="420" alt="image" src="https://github.com/user-attachments/assets/dae8f115-7ff8-454e-855e-ed1ff43c719e">
</table>

Reply:
Looking good!